src/Utils/String.ts

```diff
@@ -0,0 +1,13 @@
+const quotes = [ "'", '"' ];
+
+export function strip(input: string, chars: string[] = quotes): string {
```

Comment:
A more efficient implementation of this would be to check whether the first and the last characters are the same and whether the first character is in the set of quotes. Can you rewrite the code to use a `Set`?

Reply:
Swapping out for TypeScripts utility: https://github.com/microsoft/TypeScript/blob/387c917765793773a6f7184bab84e6f5956f44fc/src/compiler/utilities.ts#L3158-L3164

Reply:
:+1:

Reply:
In hindsight the `stripQuotes` function is internal, so it is not exported from the typescript module :(

Reply:
Ups. Let's copy the basics of the implementation then and add a comment where the code is from.